src/src/Commands/Time.cpp

```diff
@@ -54,3 +55,36 @@ String Command_DST(struct EventStruct *event, const char* Line)
 	}
 	return return_command_success();
 }
+
+String Command_SetDateTime(struct EventStruct *event, const char* Line)
+{
+	String TmpStr1;
+	if (GetArgv(Line, TmpStr1, 2)) {
+		struct tm tm;
+		int yr, mnth, d;
+		sscanf(TmpStr1.c_str(), "%4d-%2d-%2d", &yr, &mnth, &d);
+		tm.tm_year = yr - 1970;
+		tm.tm_mon = mnth;
+		tm.tm_mday = d;
+		if (GetArgv(Line, TmpStr1, 2)) {
+			int h, m, s;
+			sscanf(TmpStr1.c_str(), "%2d:%2d:%2d", &h, &m, &s);
+			tm.tm_hour = h;
+			tm.tm_min = m;
+			tm.tm_sec = s;
+		}else{
+			tm.tm_hour = 0;
+			tm.tm_min = 0;
+			tm.tm_sec = 0;
+		}
+
+		sysTime = makeTime(tm);
+		
+	}else  {
+		serialPrintln();
```

Comment:
Why do you write to the Serial port here?

Reply:
> 
> 
> Why do you write to the Serial port here?

I copied code from a similar function in Time.cpp - perhaps, that's not needed, as well in other functions in this module.

Reply:
There is only a limited number of commands (for debug purposes mainly) that output to the serial port.
For example meminfo and meminfodetail.
And I guess even those are no longer needed to output to the serial port.